lib_nbgl/src/nbgl_use_case.c

```diff
@@ -1526,5 +1547,197 @@ void nbgl_useCaseSpinner(const char *text)
     pageContext = nbgl_pageDrawSpinner(NULL, (const char *) text);
     nbgl_refreshSpecial(FULL_COLOR_PARTIAL_REFRESH);
 }
+
+#ifdef NBGL_KEYPAD
+/**
+ * @brief internal function to refresh the keypad and automatically show / hide:
+ *        - backspace key if no digits are present
+ *        - validation key if the min digit is reached
+ *        - keypad if the max number of digit is reached
+ *
+ * @param add indicate if a digit has been added or removed
+ */
+static void nbgl_refreshKeyPad(bool add)
```

Comment:
Please follow the current syntax and do not use the "refresh" word, already used for "screen refresh"... nbgl_useCaseUpdateKeypad()

Reply:
Sorry, if it's a static function, it can be have any name

Reply:
This function is `static`. I have renamed it `updateKeyPad`.